libpermian/caserunconfiguration/__init__.py

```diff
@@ -180,6 +180,35 @@ def openLogfile(self, name, mode="r", autoadd=False):
             self.addLog(name, log_path)
         return log_fo
 
+    def dropFile(self, name, data):
```

Comment:
Instead of "drop" word which sounds like "lose" if not used as a phrasal verb (like "drop in") I'd prefer probably "attach" or some other verb which sounds like the data is added.

Reply:
I see, but "attach" sounds like the file already exists.

...."create"?